src/linear_algebra/alternating.lean

```diff
@@ -407,11 +418,14 @@ where `n` is the number of inputs. -/
 lemma coe_alternatization [fintype ι] (a : alternating_map R M N' ι) :
   (↑a : multilinear_map R (λ ι, M) N').alternatization = nat.factorial (fintype.card ι) • a :=
 begin
-  ext,
-  simp only [multilinear_map.alternatization_apply, map_perm, smul_smul, ←nat.smul_def, coe_mk,
-    smul_apply, add_monoid_hom.coe_mk, finset.sum_const, coe_multilinear_map, one_smul,
-    multilinear_map.dom_dom_congr_apply, int.units_coe_mul_self,
-    finset.card_univ, fintype.card_perm],
+  apply alternating_map.coe_inj,
+  rw multilinear_map.alternatization_def,
+  simp_rw [coe_dom_dom_congr, smul_smul, int.units_coe_mul_self, one_smul,
+    finset.sum_const, finset.card_univ, fintype.card_perm, ←nat.smul_def],
+  rw [←coe_multilinear_map, coe_smul],
+  -- `nat.smul_def` introdices a `((•) : ℕ → _ → _)` diamond we have to resolve
+  rw subsingleton.elim add_comm_monoid.nat_semimodule,
+  apply_instance,
```

Comment:
This will probably be fixed by #5654, which introduces a nicer-behaving lemma than `nat.smul_def`.